android/tangram/src/com/mapzen/tangram/MapController.java

```diff
@@ -222,9 +224,29 @@ static MapController getInstance(GLSurfaceView view) {
      * @param path Location of the YAML scene file within the application assets
      */
     public void loadSceneFile(String path) {
+        loadSceneFile(path, null);
+    }
+
+    /**
+     * Load a new scene file
+     * @param path Location of the YAML scene file within the application assets
+     * @param sceneUpdates List of {@code SceneUpdate}
+     */
+    public void loadSceneFile(String path, List<SceneUpdate> sceneUpdates) {
+
+        String[] updateStrings = null;
+
+        if (sceneUpdates != null) {
+            updateStrings = new String[sceneUpdates.size() * 2];
```

Comment:
The logic to build `updateStrings` is repeated again below in `queueSceneUpdate(...)`. Could be extracted into a private method?